### DISCUSSION TITLE
503 Server Error

### DISCUSSION THREAD

**Discussion by stevenglansbergf2p** [dahlb/ha_carrier#135] [General]
Has anyone encounter a 503 server error when initializing? I seem to be getting an error of:

Service Unavailable for url: https://www.app-api.ing.carrier.com/users/authenticated

Do I need to fully reinstall the integration? Is it carrier's service being down and I should expect it to start working again in a day or two? Has Carrier removed the ability to use this integration?

I didn't want to raise an issue if this is not a problem with the integration. I've tried downgrading and reinstalling without removing the old integration. Any feedback would be nice. Thanks!

**Comment by dahlb** [comment]
just confirmed carrier's authentication api is down for me too, my assumption is maintenance window, I expect it to be fixed by tomorrow

**Reply by stevenglansbergf2p** [reply]
Thanks! I hope it starts working again soon!

**Reply by LSR19** [reply]
> Thanks! I hope it starts working again soon!

Down for me, too.

**Comment by dahlb** [comment]
Back up again
